components/financial_assistance/app/assets/javascripts/financial_assistance/income.js

```diff
@@ -1064,6 +1064,9 @@ $(document).on('turbolinks:load', function () {
     return isValid;
   }
 
+  $(document).on('focus', '.new-income-form .phone_number', function() {
```

Comment:
Context: this is the fix for https://www.pivotaltracker.com/n/projects/2640062/stories/187895118. Seems the new cloned form doesn't play nice with the masks we setup from `application.js.erb`. I tried several things, but repeating that mask here is the best I could do